azure-templates/pre-job-steps.yml

```diff
@@ -0,0 +1,52 @@
+# File: pre-job-steps.yml
+##### Steps Template Before Any Builds #####
+
+# Inputs: buildTools, customDevice, lvVersion
+parameters:
+  - name: customDevice
+    type: string
+  - name: lvVersion
+    type: string
+  - name: buildTools
+    type: string
+    default: niveristand-custom-device-build-tools
+
+
+steps:
+###########  Configure Job-wide Variables  ###########
+  # --- lvConfigVersion (.NET Assembly Version), lvPath, architecture ---
+  - ${{ if eq( parameters.lvVersion, '2020' )}}:
+    - powershell: |
+        Write-Host "##vso[task.setvariable variable=lvPath]C:\Program Files (x86)\National Instruments\LabVIEW ${{ parameters.lvVersion }}"
+        Write-Host "##vso[task.setvariable variable=architecture]x86"
+        Write-Host "##vso[task.setvariable variable=lvConfigVersion]8.0.0.0"
+      displayName: Use .NET Assembly 8.0.0.0 & 32-bit
+  - ${{ if eq( parameters.lvVersion, '2021' )}}:
+    - powershell: |
+        Write-Host "##vso[task.setvariable variable=lvPath]C:\Program Files\National Instruments\LabVIEW ${{ parameters.lvVersion }}"
+        Write-Host "##vso[task.setvariable variable=architecture]x64"
+        Write-Host "##vso[task.setvariable variable=lvConfigVersion]9.0.0.0"
+      displayName: Use .NET Assembly 9.0.0.0 & 64-bit
+  - ${{ if eq( parameters.lvVersion, '2023' )}}:
+    - powershell: |
+        Write-Host "##vso[task.setvariable variable=lvPath]C:\Program Files\National Instruments\LabVIEW ${{ parameters.lvVersion }}"
+        Write-Host "##vso[task.setvariable variable=architecture]x64"
+        Write-Host "##vso[task.setvariable variable=lvConfigVersion]10.0.0.0"
+      displayName: Use .NET Assembly 10.0.0.0 & 64-bit
```

Comment:
I understand why we're starting with this approach, but FYI that eventually we will need to allow specifying the LabVIEW year version and bitness separately and not assume that 2021 and 2023 imply 64-bit.

Reply:
If I can find a way to cleanly pass multiple parameters into an 'each' loop for the buildSteps (in the main pipeline), I imagine I could pass lvVersion and bitness in the same fashion.  Otherwise, I could pass in bitness with lvVersion and add a parameter to make these decoupled.

Reply:
bitness and lvVersion separated into different conditionals in commit 2